src/components/PersonDropdown.tsx

```diff
@@ -0,0 +1,95 @@
+import debounce from 'lodash.debounce';
+import classNames from 'classnames';
+
+import React, { useState, useCallback, useMemo } from 'react';
+
+import { Person } from '../types/Person';
+
+type Props = {
+  onSelect: (slug: string) => void,
+  people: Person[],
+  delay: number,
+};
+
+export const PersonDropdown: React.FC<Props> = ({
+  onSelect, people, delay,
+}) => {
+  const [query, setQuery] = useState('');
+  const [appliedQuery, setAppliedQuery] = useState('');
+
+  const [isDropdownOpen, setDropdownOpen] = useState(false);
+
+  const applyQuery = useCallback(
```

Comment:
To avoid this warning pass callback:
https://stackoverflow.com/questions/69830440/react-hook-usecallback-received-a-function-whose-dependencies-are-unknown-pass